### PR TITLE
ui: merge back office and front site UIs

### LINKED ISSUE #65: ui: merge main and backoffice UI in one app
Currently, main and backoffice UI are split in 2 different React apps, so authorization is handled by backend.

We can merge together the 2 apps to simplify the development and enable several features.

### PR DESCRIPTION
**Pull request by ntarocco**
* merge back office and front site UIs into one
* integrate Invenio webpack builds
* closes #65

**BLOCKED BY** releases of:
* https://github.com/inveniosoftware/invenio-i18n/pull/74
* https://github.com/inveniosoftware/invenio-assets/pull/82

### CODE REVIEW

File: invenio_app_ils/config.py [hunk 1]
```diff
@@ -123,14 +123,7 @@ def _(x):
...
 # ===================
-#: Site name
-THEME_SITENAME = _("invenio-app-ils")
-#: Use default frontpage.
-THEME_FRONTPAGE = True
-#: Frontpage title.
-THEME_FRONTPAGE_TITLE = _("invenio-app-ils")
-#: Frontpage template.
-THEME_FRONTPAGE_TEMPLATE = "invenio_app_ils/main.html"
+THEME_FRONTPAGE = False
```
Comment (ntarocco) [inline invenio_app_ils/config.py:126]: Removed `invenio-theme` views

File: invenio_app_ils/permissions.py [hunk 2]
```diff
@@ -55,7 +55,5 @@ def views_permissions_factory(action):
     """Default ILS views permissions factory."""
     if action == "circulation-loan-request":
         return authenticated_user_permission()
-    elif action == "ils-backoffice-view":
```
Comment (ntarocco) [inline invenio_app_ils/permissions.py:58]: The view is now unprotected, there is an issue open for this: https://github.com/inveniosoftware/invenio-app-ils/issues/90

File: invenio_app_ils/templates/invenio_app_ils/index.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{# -*- coding: utf-8 -*- Copyright (C) 2018 CERN. invenio-app-ils is free
```
Comment (ntarocco) [inline invenio_app_ils/templates/invenio_app_ils/index.html]: Got rid of `base`, `main` and `backoffice` jinja templates, replaced by this one.

File: scripts/bootstrap [hunk 4]
```diff
@@ -13,5 +13,5 @@ pip install -e .[all]
 
 npm update && npm install -g node-sass@4.9.0 clean-css@3.4.19 uglify-js@2.7.3 requirejs@2.2.0
 
-CWD=`pwd`
-bash "${CWD}/scripts/build_assets"
+ils collect -v
```
Comment (ntarocco) [inline scripts/bootstrap]: Changes build script

**Review by zzacharo — DISMISSED** [review]

File: invenio_app_ils/templates/invenio_app_ils/index.html [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+<html>
+  <head>
+    {%- block css %} {{ webpack['vendor.css']}} {{ webpack['app_ils.css']}} {%-
+    endblock %}
+  </head>
+
+  <body>
+    <div
+      id="invenio-config"
+      data-config="{{ ui_config | default(&quot;{}&quot;) | tojson }}"
```
[the remark this answers is not in the document]
Comment (ntarocco) [reply invenio_app_ils/templates/invenio_app_ils/index.html]: Fixed

[the remark this answers is not in the document]
Comment (ntarocco) [reply invenio_app_ils/templates/invenio_app_ils/index.html]: Fixed

**Pull request merged**